cpp/ah_wrapper/include/wrapper.h

```diff
@@ -22,6 +23,10 @@ class AHWrapper {
   int connect();
   int read_write_once(const std::array<float, 6> &cmd_values,
                       const Command &cmd, const uint8_t &reply_mode);
+  int write_once(const std::array<float, 6> &cmd_values,
+      Command cmd,
+      uint8_t reply_mode);
```

Comment:
The formatting here is weird. I'd do something like this, so it's consistent with the formatting of `read_write_once`
```suggestion
  int write_once(const std::array<float, 6> &cmd_values,
                 Command cmd, uint8_t reply_mode);
```